0.0.1/src/models.py

```diff
@@ -35,6 +35,16 @@ class MpcKey:
     account_id: str
 
 
+@dataclass
+class MultiActionSignatureRequest:
```

Comment:
Do we write these by hand or is it autogened?

Reply:
It was manual